tests/components/reolink/conftest.py

```diff
@@ -65,6 +63,13 @@ def reolink_connect(mock_get_source_ip: None) -> Generator[MagicMock, None, None
         yield host_mock
 
 
+@pytest.fixture
+def reolink_ONVIF_wait(mock_get_source_ip: None) -> Generator[MagicMock, None, None]:
```

Comment:
```suggestion
def reolink_ONVIF_wait(mock_get_source_ip: None) -> Generator[None, None, None]:
```

Reply:
Do we need the `mock_get_source_ip` fixture in this fixture?

Reply:
No I will remove it